app/utils/nonsipp/check/LoansCheckStatusUtils.scala

```diff
@@ -71,16 +71,6 @@ object LoansCheckStatusUtils {
     userAnswers: UserAnswers,
     srn: Srn,
     recordIndex: Max5000
-  ): Boolean =
-    userAnswers.get(LoanPrePopulated(srn, recordIndex)) match {
```

Comment:
Is this right @narendrapaduchuri ? It looks like you are removing the methods that are based on pre-populated flag.
I'm a bit confused with this change, could you please explain on a call?